bundles/binding/org.openhab.binding.myq/src/main/java/org/openhab/binding/myq/internal/MyqData.java

```diff
@@ -82,16 +85,14 @@ public GarageDoorData getMyqData() {
 	 *            Attempt number when it recursively calls itself
 	 */
 	private String getGarageStatus(int attemps) {
-		if (this.sercurityTokin == null && Login()) {
+		if (sercurityTokin == null && !Login()) {
```

Comment:
@digitaldan I think you are right that a timer would be redundant. I hope my retry method makes sense to to you. I just call the login method and recursively call the method again with the retry count value incremented. I don't know much about web API's but I was afraid the "User-Agent" ID will get blocked by Chamberlain if they see a bunch of login requests so I have been probably overly cautious for no good reason. Thanks for the help and have a safe trip.